.github/workflows/release.yml

```diff
@@ -24,13 +24,13 @@ jobs:
         run: |
           go build -o lambda-webhook ./cmd/lambda
           zip $ZIP_ASSET_NAME lambda-webhook
+          echo $ZIP_ASSET_NAME >> $GITHUB_ENV
```

Comment:
That's not the correct syntax I believe. To make this easier, [here's an action you can use](https://github.com/allenevans/set-env) to do this as a separate step to place before this step.

Reply:
(and make sure to test it again 😁)

Reply:
Thanks for the tip. I did not know about it.
Fixed now.